runtime/common/src/migrations.rs

```diff
@@ -203,7 +203,8 @@ where
 			// Box::new(migration_parachain_staking_purge_stale_storage),
 			// completed in runtime 1000
 			// Box::new(migration_parachain_staking_manual_exits),
-			Box::new(migration_parachain_staking_increase_max_delegations_per_candidate),
+			// completed in runtime 1101
```

Comment:
I think this one was a bit tricky because we had to revise it and run two different versions. This comment may suffice, though. I'll query our 3 main chains and see which versions are in storage.

Reply:
I was making that way too difficult (deriving storage keys manually). You can query migrations easily in developer->chain state->storage :)

Moonbase Alpha: used v2 to upgrade
Moonriver: used v2 to upgrade
Moonbeam: used v2 to upgrade

I think we caught the problem in v1 early before it ever got to Moonbase Alpha. Anyway, I've confirmed that our major networks have received it...